index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <style>
+      @media only screen and (max-width: 1200px){
+    /*Tablets [601px -> 1200px]*/
+      }
+      @media only screen and (max-width: 600px){
+        /*Big smartphones [426px -> 600px]*/
+      }
+      @media only screen and (max-width: 425px){
+        /*Small smartphones [325px -> 425px]*/
+      }
+
+      .flex-container {
+      display: flex;
+    }
+
+    header {
+  position: fixed;
+  width: 100%;
+  top: 0;
+  left: 0;
+}
+
+    </style>
```

Comment:
For best performance of your project kindly separate your css cods from html codes.

Reply:
- [ ] Kindly consider using suggestions instead of orders, so your comment appears more polite